lib/modules/datasource/nuget/index.spec.ts

```diff
@@ -243,6 +244,70 @@ describe('modules/datasource/nuget/index', () => {
       expect(res).toBeNull();
     });
 
+    it('logs instead of triggering a TypeError when PackageBaseAddress is missing from service index', async () => {
+      const nugetIndex = `
+        {
+          "version": "3.0.0",
+          "resources": [
+            {
+              "@id": "https://api.nuget.org/v3/metadata",
+              "@type": "RegistrationsBaseUrl",
+              "comment": "Get package metadata."
+            },
+            {
+              "@id": "https://api.nuget.org/v3/metadata",
+              "@type": "RegistrationsBaseUrl/3.0.0-beta",
```

Comment:
I think we only need one of these, so remove one

Reply:
I was surprised, but the test doesn't pass without both.

Reply:
Ah, I tried getting rid of the versioned one, which failed. Removing the other one does the trick.